build.sbt

```diff
@@ -10,6 +10,7 @@ lazy val root = (project in file("."))
   .enablePlugins(PlayScala, SbtAutoBuildPlugin, SbtDistributablesPlugin, SbtArtifactory)
```

Comment:
disable junitreporter. There is an example in cbcr as the tests will fail in jenkins because of that